server/lib/emailTemplates.js

```diff
@@ -24,6 +24,7 @@ const templateNames = [
   'group.donation.created',
   'group.monthlyreport',
   'group.monthlyreport.text',
+  'onetime',
```

Comment:
Please add a `onetime.text` template with the text version.

Also isn't `announcement` better suited here as a name for the template? (not a strong opinion)